default.json

```diff
@@ -37,6 +37,12 @@
       "allowedVersions": "< 0.15.0",
       "enabled": true
     },
+    {
+      "groupName": "mergo",
+      "matchPackagePatterns": ["^github.com/imdario/mergo"],
+      "allowedVersions": "< 1.0.0",
+      "enabled": true
+    },
     {
       "groupName": "misc",
       "matchPackagePatterns": ["^github.com/operator-framework/api", "^github.com/ghodss", "^github.com/go-logr/logr", "^github.com/imdario/mergo", "^go.uber.org/zap"],
```

Comment:
should we remove mergo from this line as part of this change?

Reply:
good point